src/main/java/edu/umich/its/cpm/EmailFormatter.java

```diff
@@ -43,15 +42,81 @@ public EmailFormatter(String emailMgs, AttachmentHandler attachmentHandler) thro
     }
 
     public String rfc822Format() {
-        StringBuilder emailInRFCFormat = new StringBuilder();
         ArrayList<String> headers = prunedHeadersWithoutContentType();
+        return getFormattedEmailText(headers).toString();
+    }
+
+    /*
+      Mbox Format standard is defined by RFC4155 document and extends RFC2822 Standard.
+      Mbox format email messages file must start like "From doe@eg.edu Wed Aug 24 14:04:47 2016" and
+      each message is separated by blank line
+     */
+    public String mboxFormat() {
+        ArrayList<String> headers = mboxFormatHeaders();
+        return getMboxFormattedEmailText(headers);
+    }
+
+    private StringBuilder getFormattedEmailText(ArrayList<String> headers) {
+        StringBuilder emailFormat = new StringBuilder();
         for (String header : headers) {
-            emailInRFCFormat.append(header);
-            emailInRFCFormat.append("\r\n");
+            emailFormat.append(header);
+            emailFormat.append("\r\n");
         }
         String bodyAndAttachment = emailTextWithBodyAndAttachment(getEmailText());
-        emailInRFCFormat.append(bodyAndAttachment);
-        return emailInRFCFormat.toString();
+        emailFormat.append(bodyAndAttachment);
+        emailFormat.append("\r\n");
+        return emailFormat;
+    }
+
+    private String getMboxFormattedEmailText(ArrayList<String> headers) {
+        StringBuilder emailFormat = new StringBuilder();
+        for (String header : headers) {
+            emailFormat.append(header);
+            emailFormat.append("\r\n");
+        }
+        String bodyAndAttachment = emailTextWithBodyAndAttachment(getEmailText());
+        ArrayList<String> pureBodyText = mboxBodyFormatting(bodyAndAttachment);
+        String[] split = bodyAndAttachment.split("\r\n");
+        List<String> formattedBodyAndAttachment = Arrays.asList(split);
+        for (int i=0;i<4;i++) {
```

Comment:
1.Content-Type: multipart/mixed; 
2.	boundary="----=_Part_0_1265915532.1473784893296"
3.
4. ------=_Part_0_1265915532.1473784893296
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 7bit

xxx
yyy
zzz


------=_Part_0_1265915532.1473784893296--

I am appending the first 4 in the above string `pureBodyText` . this is part of string manipulation if  message body is starting `From ` -> `>From`